content/general/push/_push_intro.textile

```diff
@@ -0,0 +1,45 @@
+
+h2(#deliver). Delivering push notifications
+
+Ably provides two models for delivering push notifications to devices:
+
+h3(#channel-broadcasting). Channel-based broadcasting
+
+The model for delivering push notifications to devices over channels is intentionally very similar to how messages are normally delivered using Ably's "pub/sub channel":/core-features/channels. For example, a normal message published on an Ably channel is broadcast immediately to all realtime subscribers of that channel. When broadcasting push notifications on channels, however, the process is the same with the exception that the subscribers (devices receiving push notifications) are registered in advance using our API and the message itself must contain an *extra push notification payload* that specifies the optional visual format and optional data payload of the native push notification.
+
+"Find out more about channel-based push notification broadcasting":/general/push/publish#channel-broadcast
+
+h3(#direct-publishing). Direct publishing
+
+Ably provides a REST API that allows native push notifications to be delivered directly to:
+
+* Devices identified by their unique device ID
+* Devices identified by their assigned "@clientId@":/realtime/authentication#identified-clients
+* Devices identified by the recipient details of the native push transport such as their unique @registrationToken@ in the case of FCM, @deviceToken@ in the case of APNS, or @targetUrl@ and @encryptionKey@ in the case of a Web device. This means is particularly useful when migrating to Ably with existing push notification target devices.
+
+"Find out more about direct push notification publishing":/general/push/publish#direct-publishing
+
+h2(#activate). Activating a device and receiving notifications
+
+Every device that will receive push notifications must activate itself with the local operating system or framework, and hook into the push notification services that the underlying platform provides. This functionality is platform-specific and can also vary considerably across not just platforms, but also across the push services that operate on those platforms such as GCM and FCM, both of which are available on the Android platform.
+
+The Ably client libraries aim to abstract away this complexity and platform-specific behaviour by providing a consistent API for device activation, maintenance of the device registration, subscription to Ably channels for receiving push notifications, and of course for receiving the native push notifications themselves.
```

Comment:
We don't abstract "receiving the native push notifications themselves". Apps will still use the existing native platform APIs for this.